app/components/support_interface/application_summary_component.rb

```diff
@@ -178,6 +180,47 @@ def subscribed_to_emails?
       candidate.subscribed_to_emails?
     end
 
+    def find_a_candidate_state_row
+      {
+        key: 'Find a Candidate opt-in status',
+        value: if application_form_in_the_pool?
+                 'Currently in the candidate pool'
+               elsif candidate.published_opt_in_preferences.present?
+                 'Opted in (not in pool)'
```

Comment:
I feel like we should move away from calling it a pool, even in support, since we don't expose that language to users. Maybe something like `Currently findable by providers` and `Opted in (not findable)` would be more consistent with the language candidates and providers user?

Also, can we put the strings in a locale file?